src/js/game_logic.js

```diff
@@ -44,21 +47,32 @@ const correctAnswer = () => {
         countElement.className = 'rhyme_count'
         guessElement.className = ''
         if (Game.currentCard.id < puzzles.length) {
-                loadCard(puzzles[Game.currentCard.id])
+            loadCard(puzzles[Game.currentCard.id])
         }
         else gameOver()
     }, 800)
 }
 
 const incorrectAnswer = () => {
+
     cardBlock.className += ' incorrect'
     countElement.className += ' incorrect'
     guessElement.className += 'incorrect'
+    numOfincorrectanswers++
+
+
     setTimeout(() => {
         cardBlock.className = 'rhyme_card'
         countElement.className = 'rhyme_count'
         guessElement.className = ''
     }, 500)
+    const wronganswers = document.getElementById('incorrect')
```

Comment:
Let's match the naming convention here and use wrongElement or wrongAnswersElement. Any time you use getElement code, the variable name should include 'Element' as well.